BasementRenovator.py

```diff
@@ -6300,9 +6300,17 @@ def setup(modPath, roomsPath, floorInfo, rooms, version):
                     "",
                 )
 
+            # Prefix the file path with the root that Proton uses
+            # Only necessary for Repentance and Repentance+ as they don't have native Linux support.
+            if (
+                version in ["Repentance", "Repentance+"]
+                and "Linux" in platform.system()
+            ):
+                modPath = "Z:\\" + modPath
```

Comment:
Does this work? The PureWindowsPath should fix this up anyway
```suggestion
                modPath = "Z:/" + modPath
```

Reply:
Yep, seems to work just fine